### PR TITLE
fix(CX-2138): login and signup screen, bottom buttons dont obey SafeArea

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Bugfix**

This PR resolves [CX-2138]

### Description

Removed <ArtsyKeyboardAvoidingView/> that wrapped NavigationContainer of all Onboarding screens (file Onboarding.tsx).  Instead placed it inside the <View/> wrapper for each component of Onboarding flow. To the mentioned View added paddingButtom: 10 (used 10 because we use paddingBottom: 20 on OnboardingWelcome screen for the text and 10 looks ok for the button. I couldn't find the design for these screens in figma, looks like they've been updated recently)

After fixes. On the left - Android, on the right - iOS
https://user-images.githubusercontent.com/36167539/157866769-42f35947-f5b2-4def-8752-85c1b5b8f997.mov

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- small design improvements for the onboarding screens -daria

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-2138]: https://artsyproduct.atlassian.net/browse/CX-2138?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (small design improvements for the onboarding screens -daria)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 66e3b90b5eca4c351aac86510a6dcc0089c57f66
</p>

File: src/app/Scenes/Onboarding/OnboardingCreateAccount/OnboardingCreateAccount.tsx [hunk 1]
```diff
@@ -153,42 +154,49 @@ export const OnboardingCreateAccountWithEmail: React.FC<OnboardingCreateAccountP
...
-          navigateToLoginWithEmail={() => {
-            navigation.replace("OnboardingLoginWithEmail", {
-              withFadeAnimation: true,
-              email: formik.values.email,
-            })
-          }}
-        />
-      </NavigationContainer>
-    </FormikProvider>
+    <View style={{ flex: 1, backgroundColor: "white", flexGrow: 1, paddingBottom: 10 }}>
```
Comment (pvinis) [inline src/app/Scenes/Onboarding/OnboardingCreateAccount/OnboardingCreateAccount.tsx:157]:
```suggestion
    <Flex flex={1} backgroundColor="white" flexGrow={1} paddingBottom={10}>
```
how about this 🤔?

Comment (pvinis) [reply src/app/Scenes/Onboarding/OnboardingCreateAccount/OnboardingCreateAccount.tsx:157]: also, are `flexGrow` and `paddingBottom` both needed?

**Review by pvinis — APPROVED** [review]
Very nice! I think it might be nicer to use `Flex`, but it's not a blocking thing. also maybe flexGrow is not needed? I'll assign back to you, and merge when you want 👍

**Comment by dariakoko** [comment]
> Very nice! I think it might be nicer to use `Flex`, but it's not a blocking thing. also maybe flexGrow is not needed? I'll assign back to you, and merge when you want 👍

Thanks Pavlos! Replaced <View/> with <Flex/> 👍  I would keep flexGrow and paddingBottom, looks like they both needed. When I remove flexGrow the whole screen is sliding to the very top (🙄) and paddingBottom is needed for the bottom padding

**Comment by pvinis** [comment]
thanks for looking into it!

**Comment by dariakoko** [comment]
> thanks for looking into it!

How are you so fast ahahahah

**Pull request merged**